packages/sdk/src/client.ts

```diff
@@ -316,7 +320,12 @@ export class Client
         this.syncedStreamsExtensions.setStreamIds(streamIds)
     }
 
-    async initializeUser(newUserMetadata?: { spaceId: Uint8Array | string }): Promise<void> {
+    async initializeUser(
+        opts?: {
+            metadata?: { spaceId: Uint8Array | string }
+        } & EncryptionDeviceInitOpts,
+    ): Promise<void> {
```

Comment:
just change `newUserMetadata?: { spaceId: Uint8Array | string }`
to `opts?: { spaceId: Uint8Array | string, encryptionDevice?: EncryptionDeviceInitOpts }`

Reply:
and delete line 331